.eslintrc

```diff
@@ -24,7 +24,7 @@
   "rules": {
     "oppia/break-after-parens": "error",
     "oppia/test-message-style": "error",
-    "oppia/no-unused-dependency": "error",
+    "oppia/dependency-checks": "error",
```

Comment:
Why this change?

Reply:
This file has two rules in it, both belong to the dependency injection. So changing the name to dependency-checks.